api/pkg/cli/cmd/search/search.go

```diff
@@ -156,7 +160,7 @@ func (opts *options) run() error {
 func (opts *options) validate() error {
 
 	if flag.AllEmpty(opts.args, opts.kinds, opts.tags, opts.categories) {
```

Comment:
Could you please add platforms here as well to check emptiness of platforms flag
```suggestion
	if flag.AllEmpty(opts.args, opts.kinds, opts.tags, opts.categories,opts.platforms) {
```